src/app/feed/feed-post-dropdown/feed-post-dropdown.component.ts

```diff
@@ -25,7 +25,8 @@ export class FeedPostDropdownComponent {
     private platformLocation: PlatformLocation
   ) {}
 
-  reportPost() {
+  reportPost(): void {
+    this.globalVars.logEvent("report-content");
```

Comment:
Pls follow the syntax `singular object : present tense verb : extra context` so perhaps `post: report`